packages/server/lib/plugins/child/run_require_async_child.js

```diff
@@ -124,16 +123,10 @@ function run (ipc, file, projectRoot) {
         debug(`User doesn't have esbuild. Going to use native node imports.`)
 
         // We cannot replace the initial `require` with `await import` because
-        // Certain modules cannot be dynamically imported. If this throws, however, we want
-        // to show the original error that was thrown, because that's ultimately the source of the problem
-        try {
-          return await import(file)
-        } catch (e) {
-          // If we aren't able to import the file at all, throw the original error, since that has more accurate information
-          // of what failed to begin with
-          debug('esbuild fallback for loading config failed, throwing original error. node import error: %o', e)
-          throw originalError
-        }
+        // Certain modules cannot be dynamically imported.
+
+        // pathToFileURL for windows interop: https://github.com/nodejs/node/issues/31710
+        return await import(pathToFileURL(file).href)
```

Comment:
If the file is a `.ts` file, we will get a problem, though, since native imports do not support `.ts`.
https://github.com/cypress-io/cypress/issues/22071
I will look into it.